setup.py

```diff
@@ -215,7 +215,7 @@ def run(self):
             print('-- Not using CUDA')
         if WITH_NCCL and WITH_SYSTEM_NCCL:
             print('-- Using system provided NCCL library at ' +
-                  NCCL_LIB_DIR + ', ' + NCCL_INCLUDE_DIR)
+                  NCCL_SYSTEM_LIB + ', ' + NCCL_INCLUDE_DIR)
```

Comment:
maybe include lib dir as well?

Reply:
that's redundant information, i dont think we need it.

Reply:
No. It's important. You might have multiple versions of nccl installed (e.g. one in our lib dir and one system, and you want to verify that it picked up the one you wanted).

Reply:
NCCL_SYSTEM_LIB is the full path to the library, it includes NCCL_LIB_DIR as it's prefix